app/routes.php

```diff
@@ -16,5 +16,7 @@
 
     $app->post('/products', 'AddProductController');
 
+    $app->put('/products/stock/{sku}', 'UpdateProductStockController');
     $app->get('/products', 'GetProductsController');
```

Comment:
Maybe organize this? groups /products then products/{sku} then products/stock/{sku} ?